build.gradle

```diff
@@ -0,0 +1,65 @@
+plugins {
+    id 'java'
+    id 'application'
+}
+
+version '1.0'
+
+repositories {
+    mavenCentral()
+}
```

Comment:
May be there is a repo for Eclipse EMF jars.

Reply:
I have looked thouroughly and couldn't find anything. A couple of resources lead me to believe that Eclipse simply does not maintain a Maven repo for their EMF jars. There is this https://repo.eclipse.org/content/, but it only has the older versions just like Maven Central.

Reply:
ok, anyway we can improve it later if we find a way.